### DISCUSSION TITLE
Move detailed testing instructions to tests/readme.md?

### DISCUSSION THREAD

**Discussion by haipinglu** [pykale/pykale#63] [Testing] [answered]
@bobturneruk Now the testing is about 100 lines (1/3 of `CONTRIBUTING.md`). How about keeping a brief version there and putting the details at `tests/README.md`? At this location, we can add even more details if needed.

**Comment by bobturneruk** [comment, marked as answer]
Good idea! I'll raise an issue and assign myself.
